### DISCUSSION TITLE
Doubt regarding `forge test`

### DISCUSSION THREAD

**Discussion by Jaseempk** [Cyfrin/foundry-full-course-cu#346] [Q&A] [answered]
`forge test --match-test <fn-name>`  This command is running my test when I use the same function name as Patrick used in his code, but when I use a random name for the Test Function it's not working, so is there any predefined naming system for this or I'm making mistake in something else?

![Screenshot 2023-07-15 122549](https://github.com/Cyfrin/foundry-full-course-f23/assets/99121494/3cc5a321-5741-4c8c-b963-7e6c3692e5f7)

**Comment by shikhar229169** [comment, marked as answer]
Actually Foundry picks only those functions for test if they have the prefix 'test'

**Reply by Jaseempk** [reply]
Yea got it, I read the documentation..thanks
